### PR TITLE
[All] (Mac) Add info about clear web cache option

### PR DESCRIPTION
**Pull request by davidchesnut**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[85055b0](https://github.com/OfficeDev/office-js-docs-pr/pull/1052/commits/85055b0c34c356117a0de4e25d678af18daf56dd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/mac-clear-cache-menu.png](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/images/mac-clear-cache-menu.png) | :white_check_mark:Succeeded | |
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906032253393125-1052%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=fVOv0kEZX9Kv5SvlBrXjG8BzTsXui7qEOt82iwxn964%3D&st=2019-06-03T22%3A49%3A39Z&se=2019-07-04T22%3A54%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/debug-office-add-ins-on-ipad-and-mac.md [hunk 1]
```diff
@@ -38,6 +38,13 @@ Then, open the Office application and [sideload your add-in](sideload-an-office-
 
 Add-ins are often cached in Office for Mac, for performance reasons. Normally, the cache is cleared by reloading the add-in. If more than one add-in exists in the same document, the process of automatically clearing the cache on reload might not be reliable.
 
-On a Mac, you can clear the cache manually by deleting the contents of the `~/Library/Containers/com.Microsoft.OsfWebHost/Data/` folder. 
+You can clear the cache using an existing add-in.
```
[the remark this answers is not in the document]
Comment (mscharlock) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: Also, I think the intro blurb to this article is a little confusing. Maybe change to: 
Because add-ins are developed using HTML and JavaScript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on a Mac.
>[!NOTE] 
>You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on a Mac.

Comment (kbrandl) [inline docs/testing/debug-office-add-ins-on-ipad-and-mac.md]:
```suggestion
You can clear the cache by using the personality menu of any task pane add-in.
```

[the remark this answers is not in the document]
Comment (davidchesnut) [reply docs/testing/debug-office-add-ins-on-ipad-and-mac.md]: @mscharlock Thanks! I updated the intro and the date!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5950937](https://github.com/OfficeDev/office-js-docs-pr/pull/1052/commits/595093704fdb1978d26dad6061489c3a8db06453)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/mac-clear-cache-menu.png](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/images/mac-clear-cache-menu.png) | :white_check_mark:Succeeded | |
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907051950023602-1052%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=iI11YirKYMS7Yn3GU7vTcWsXyLbeluv73Uzx3FXn5g4%3D&st=2019-07-05T19%3A46%3A00Z&se=2019-08-05T19%3A51%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[583dad2](https://github.com/OfficeDev/office-js-docs-pr/pull/1052/commits/583dad2e9c11f3540903a730d1801ad505d4c96e)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-80de4d25)

<a id="80de4d25"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\tjri-s, target branch: master, target branch commit: d869c7e47ed8ac1bd27ae2dffacfe48623b6abd2, source commit: 583dad2e9c11f3540903a730d1801ad505d4c96e.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907051952477294-1052%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FvxjpdZNrPAR%2BWVhrvJR2EotOFQUV3wrnOtUpS0dwlo%3D&st=2019-07-05T19%3A47%3A59Z&se=2019-08-05T19%3A52%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c14c5dd](https://github.com/OfficeDev/office-js-docs-pr/pull/1052/commits/c14c5ddb2cb8070515d708dca25bb27f5345c2c8)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-dc40b1ec)

<a id="dc40b1ec"></a>
###
 - **[Error]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\3h3j-s, target branch: master, target branch commit: d869c7e47ed8ac1bd27ae2dffacfe48623b6abd2, source commit: c14c5ddb2cb8070515d708dca25bb27f5345c2c8.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907052004005443-1052%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=LqM2RDkPqOoIXsSsGRBVfuVOIiredPsVxjmrz1hvqG8%3D&st=2019-07-05T19%3A59%3A14Z&se=2019-08-05T20%3A04%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by davidchesnut** [comment]
@kbrandl I moved the clear cache instructions into the include file. All the articles that include it are rebuilt. Take a look and see if that makes sense. I suppose it is a bit wordy, but it is complete in all articles!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2c298f2](https://github.com/OfficeDev/office-js-docs-pr/pull/1052/commits/2c298f248677d2c16955bd54151310dc0cf5a236)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/mac-clear-cache-menu.png](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/images/mac-clear-cache-menu.png) | :white_check_mark:Succeeded | |
[docs/includes/mac-cache-folders.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/includes/mac-cache-folders.md) | :white_check_mark:Succeeded | |
[docs/testing/debug-office-add-ins-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/testing/debug-office-add-ins-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |
[docs/testing/sideload-an-office-add-in-on-ipad-and-mac.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/testing/sideload-an-office-add-in-on-ipad-and-mac.md) | :white_check_mark:Succeeded | |
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/davech-mac/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907291719305538-1052%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=jJuDLue41i%2FrJTrTIhI8t5Q1NtSJhUBWdl%2FPlKSf4N8%3D&st=2019-07-29T17%3A15%3A38Z&se=2019-08-29T17%3A20%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
